go.mod

```diff
@@ -0,0 +1,10 @@
+module gsclient-go
```

Comment:
should not be the full path e.g, `github.com/gridscale/gsclient-go`?

Reply:
Again, go.mod file is automatically generated by command `go mod init`

Reply:
Thanks @nvthongswansea for the clarification.